tests/simple_server.py

```diff
@@ -70,6 +70,13 @@ def log_request(self, code='-', size='-'):
 # server on those Windows/Py2 to not fill the buffer.
 if six.PY2 and platform.system() == 'Windows':
   handler = QuietHTTPRequestHandler
+
+use_quiet_http_request_handler = True
+if len(sys.argv) > 2:
+  use_quiet_http_request_handler = sys.argv[2]
+
+if use_quiet_http_request_handler == True:
```

Comment:
As this is Python and we expect a boolean value we can just test the variable like:

```suggestion
if use_quiet_http_request_handler:
```

See PEP 8's [Programming Recommendations](https://www.python.org/dev/peps/pep-0008/#id51) and the [Book of Python Anti-Patterns](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html)

Reply:
Yea, I just wanted to be more explicit, but you are right. 
I will address this.